src/Orc.Controls/Orc.Controls.Shared/Controls/DateTimePicker/Views/DateTimePickerControl.xaml.cs

```diff
@@ -45,18 +50,36 @@ public DateTimePickerControl()
                 NumericTBMinute,
                 NumericTBSecond,
             };
+        }
 
-            NumericTBDay.RightBoundReached += NumericTextBoxOnRightBoundReached;
-            NumericTBMonth.RightBoundReached += NumericTextBoxOnRightBoundReached;
-            NumericTBYear.RightBoundReached += NumericTextBoxOnRightBoundReached;
-            NumericTBHour.RightBoundReached += NumericTextBoxOnRightBoundReached;
-            NumericTBMinute.RightBoundReached += NumericTextBoxOnRightBoundReached;
+        private void SubscribeNumericTextBoxes()
+        {
+            _numericTextBoxes[0].RightBoundReached += NumericTextBoxOnRightBoundReached;
```

Comment:
I did notice, but I had to look twice to see it, hence the reason I want a comment that next time, when I am probably in a hurry, I won't destroy this logic.